### PR TITLE
DOC-9580 - rebalance button active

### PR DESCRIPTION
**Pull request by sreekanth-cb**
Updating the rebalance documentations for the active rebalance
button issue with the contexts and work arounds.

### CODE REVIEW

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 1]
```diff
@@ -62,3 +62,22 @@ The index partitions residing on the recovered node would be reused during a rec
 3. Perform the software update or hardware maintenance operations.
 4. Perform recover rebalance operation.
 5. Cluster is back to normal/pre-failover safe state.
+
+=== Why rebalance button remains active even after a successful rebalance
+
+Even after a succesful Search rebalance operation, if the rebalance button remains `active` then it hints that the Search cluster is not having enough nodes
```
Comment (Thejas-bhat) [inline modules/fts/pages/fts-rebalance-failover.adoc]:
```suggestion
Even after a successful Search rebalance operation, if the rebalance button remains `active` then it hints that the Search cluster is not having enough nodes
```

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 2]
```diff
@@ -62,3 +62,22 @@ The index partitions residing on the recovered node would be reused during a rec
...
+Though validation checks ensures that there are enough search nodes to support the configured number of `replica` partitions, the number of search nodes
+could get compromised during the lifespan of a cluster due to various unforeseen events like hardware/network failures. And the `active` rebalance button issue surface if the subsequent recovery or node addition rebalance operations for Search service isn't bringing in enough nodes to support the configured replica count
+for the partitions.
+
+The user could resolve this by either,
+
+- adding enough Search nodes to the cluster to support the configured `replica` count.
+- editing the index definition(s) to adjust the replica count plausible in the cluster.
+
+Note - The maximum number of replica count feasible in a cluster is one less than the total number of Search nodes in the cluster.
```
Comment (Thejas-bhat) [inline modules/fts/pages/fts-rebalance-failover.adoc]: The preview of the note on line 81 is not rendered properly like the first note that's there in the same file.

**Review by Thejas-bhat — APPROVED** [review]

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 3]
```diff
@@ -62,3 +62,21 @@ The index partitions residing on the recovered node would be reused during a rec
 3. Perform the software update or hardware maintenance operations.
 4. Perform recover rebalance operation.
 5. Cluster is back to normal/pre-failover safe state.
+
+=== Why rebalance button remains active even after a successful rebalance
+
+Even after a successful Search rebalance operation, if the rebalance button remains `active` then it hints that the Search cluster is not having enough nodes
+to support the number of replicas configured for the Search index. The Search service considers the cluster is still imbalanced as it isn't having enough
+hardware resources to support the set configuration of replica count for the index partitions.
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-rebalance-failover.adoc]: I'd recommend a re-phrase here ..

..if the rebalance button remains `active` then it is indicative of the cluster not containing enough nodes that host the _Search_ service to support the configured number of replicas for the index.

Also, I'd drop the 2nd sentence which feel repetitive.

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 4]
```diff
@@ -62,3 +62,21 @@ The index partitions residing on the recovered node would be reused during a rec
...
+
+Though validation checks ensures that there are enough search nodes to support the configured number of `replica` partitions, the number of search nodes
+could get compromised during the lifespan of a cluster due to various unforeseen events like hardware/network failures. And the `active` rebalance button issue surface if the subsequent recovery or node addition rebalance operations for Search service isn't bringing in enough nodes to support the configured replica count
+for the partitions.
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-rebalance-failover.adoc:72]: Typo .. surface => surfaces

File: modules/fts/pages/fts-rebalance-failover.adoc (hunk 3 above)
Comment (sreekanth-cb) [reply modules/fts/pages/fts-rebalance-failover.adoc]: 👍

File: modules/fts/pages/fts-rebalance-failover.adoc (hunk 4 above)
Comment (sreekanth-cb) [reply modules/fts/pages/fts-rebalance-failover.adoc:72]: done!

**Review by abhinavdangeti — APPROVED** [review]
Looks good.

**Review by jon-strabala — APPROVED** [review]
LGTM

File: modules/fts/pages/fts-rebalance-failover.adoc [hunk 5]
```diff
@@ -42,7 +42,7 @@ Checking the current value for `maxConcurrentPartitionMovesPerNode` in a cluster
 curl -XGET -u <username>:<password> http://<nodeIP>:8094/api/manager
 ----
 
-Note: When multiple partitions are built parallel, it needs more RAM and mandates a higher RAM quota. As the rebalance operations consume resources, it is always advisable to plan them during non-peak hours. One way to speed up the rebalance operation is to enable the movement of partitions in parallell in a configurable way.
+Note: When multiple partitions are built parallel, it needs more RAM and mandates a higher RAM quota. As the rebalance operations consume resources, it is always advisable to plan them during non-peak hours. One way to speed up the rebalance operation is to enable the movement of partitions in parallel in a configurable way.
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-rebalance-failover.adoc]: What does `.. in a configurable way` mean here.

Feel the entire sentence `One way to speed up the rebalance operation is to enable the movement of partitions in parallel in a configurable way.` should just be removed.

Comment (sreekanth-cb) [reply modules/fts/pages/fts-rebalance-failover.adoc]: good spot! 
let me reword it!

**Pull request merged**